Utilities/Hash.cpp

```diff
@@ -0,0 +1,89 @@
+#include "stdafx.h"
+#include "Hash.h"
+#include <sstream>
+#include <iomanip>
+
+#include "../rpcs3/Crypto/sha1.h"
+#include <iostream>
+namespace hash
+{
+
+std::string SHA1(const std::string& str)
+{
+	int len = str.size();
+
+	unsigned char hash[20];
+
+	std::unique_ptr<unsigned char[]> input(new unsigned char[len+1]);
+	::memcpy(input.get(), str.c_str(), len);
+
+	sha1(input.get(), len, hash);
+
+	std::stringstream res;
```

Comment:
Can be done without stringstream and shit...
```C++
fmt::format("0x%016llx", reinterpret_cast<le_t<u64>&>(hash[0]));
```
Only 16 characters but it's enough.